scs2-examples/src/test/java/us/ihmc/scs2/examples/urdf/SimpleCrossFourBarRobotTest.java

```diff
@@ -52,7 +52,7 @@ public void testLoadingURDF() throws JAXBException, IOException
 
       //      System.out.println(new String(byteArrayOutputStream.toByteArray()));
 
-      URDFModel copyURDFModel = URDFTools.loadURDFModel(new ByteArrayInputStream(byteArrayOutputStream.toByteArray()),
+      URDFModel copyURDFModel = URDFTools.loadURDFModel(Collections.singletonList(new ByteArrayInputStream(byteArrayOutputStream.toByteArray())),
```

Comment:
Shouldn't be needing this change right?

Reply:
Explained in another comment, either add new constructor, or have this change...